### ISSUE TITLE
Hide/Show unload & reload the tabs instead of masking them.

### ISSUE THREAD

**Issue by nbp** [mozilla/testpilot-containers#802]
When hiding tabs, they are closed and not hidden.  This cause additional issues which are that tabs are not restored in their original positions and that all tabs are reloaded when they are unhidden.

**Comment by MathieuDomingo** [comment]
(It's not "exactly" the same problem, but I thinks it's close enough to add this here) 

When I close Firefox and open it, it's loading only the last tab used and so it connect me only to one website (I usually open a new tab before closing to not being connect to anything at start).
With this plugin : 
- When I hide tabs, it's hiding tabs but it's also unloading all tabs / disconnecting from website.
- When I show tabs, it's loading all tabs and so connect me to all websites in these tabs. 
I would like to have at least an option to allow me to show them back in firefox, but **not** load tabs content and so **not** connect to all websites in these tabs.


I think the best should be a combination of options , something like this : 
- 1 option to allow us to choose if we only want to hide/show tabs and stay connected to websites (and stay disconnected from not connected tabs), or if we want to hide/show tabs but also being disconnected to websites
- 1 option to choose for when showing tabs if we want to reconnect all tabs to websites (even the disconnected one current default behavior), or if we want to reconnect only previously connected tab to websites [this one allow (for example) to connect to 2 tabs, close firefox, run firefox (so these 2 tabs are disconnected), open 3 new tabs and connect to website, hide tabs, show tabs and get only these 3 tabs connected to website], or if we want to not reconnect to any websites (i think this case make sense only with the 2nd case of the first option)

**Comment by ctorgalson** [comment]
Adding a word because I forgot and tried to use Multi Account Containers with Tab Groups again ([previous Bugzilla report](https://bugzilla.mozilla.org/show_bug.cgi?id=1963603) points here).

TL;DR: along with other tab metadata mentioned above, tab groups are lost when tabs are hidden.
